### PR TITLE
Document how to disable signature checking

### PR DESCRIPTION
**Pull request by theosanderson**
For unofficial updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Submitted pull requests should be made up of commits with messages that are suit
 
 The outputs of the build are
 
-- A system image, ``output/images/ot2-system.zip``, good for doing a software update
+- A system image, ``output/images/ot2-system.zip``, good for doing a software update. (If your build is unsigned you will need to disable signature checking in the robot's update server config (``/var/lib/otupdate/config.json``) and restart.)
```
Comment (sfoster1) [inline README.md]: you can actually just restart the component that handles updates, the [update server](https://github.com/Opentrons/opentrons/tree/edge/update-server): `systemctl restart opentrons-update-server`

**Review by sfoster1 — APPROVED** [review]
This is in fact the intended way to disable updates, yes. The signature checking on updates is for your protection, not to lock down the system - it's completely fine to disable it. Note that hash checking to ensure the update files are not corrupted in download is still done when you change this setting, it's just the signature part that's removed.

File: README.md (hunk 1 above)
Comment (theosanderson) [reply README.md]: Thanks -- done

**Comment by sfoster1** [comment]
Thanks @theosanderson!

**Pull request merged**